frankielb/frankie-barrios-ux-aop
# Unit 2.3 javascript

## What It Does
This pull request is set to complete Unit 2.3 of the ux-onboarding-project.

This work is taking place to complete unit 2.3 of the ux-onboarding-project.

**Implementation:**
- Added javascript to User Profile for expanding and collapsing User Profile section.
- Added additional styling changes.

[closes #11 ](https://github.com/frankielb/frankie-barrios-ux-aop/issues/9)

## How To Test
- Make sure JavaScript is enabled in your browser.
- Click on buttons on the 'User Profile' page to expand and/ or collapse the user profile card.

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 

Browsers tested:
**Many items in list N/A** 

## Dependencies
1. Install Node and npm
2. Install Yarn
